templates/3dsmax/scripts/3dsmax.ps1

```diff
@@ -80,39 +71,7 @@ function SetupDistributedRendering
 
     $script:pre_render_script_content += "if index != 1 then (print ""VRay renderer not used, please save the scene with a VRay renderer selected."")`r`n"
 
-    If($maxVersion -eq "2018")
-    {
-        IF($renderer -eq "VRayRT")
-        {
-            $script:pre_render_script_content += "index = findString rendererName ""V_Ray_RT_""`r`n"
-            $script:pre_render_script_content += "if index == 1 then (r.distributed_rendering = true)`r`n"
-            $script:pre_render_script_content += "r.V_Ray_settings.system_vrayLog_level = 4`r`n"
-            $script:pre_render_script_content += "r.V_Ray_settings.system_vrayLog_file = ""$vrayLogFile""`r`n"
-        }
-        ElseIf($renderer -eq "VRayAdv")
-        {
-            $script:pre_render_script_content += "index = findString rendererName ""V_Ray_Adv""`r`n"
-            $script:pre_render_script_content += "if index == 1 then (r.system_distributedRender = true)`r`n"
-            $script:pre_render_script_content += "r.system_vrayLog_level = 4`r`n"
-            $script:pre_render_script_content += "r.system_vrayLog_file = ""$vrayLogFile""`r`n"
-        }
-    }
-    ElseIf($maxVersion -eq "2019"){
-        IF($renderer -eq "VRayRT"){
-            $script:pre_render_script_content += "index = findString rendererName ""V_Ray_GPU_""`r`n"
-            $script:pre_render_script_content += "if index == 1 then (r.distributed_rendering = true)`r`n"            
-            $script:pre_render_script_content += "r.V_Ray_settings.system_vrayLog_level = 4`r`n"
-            $script:pre_render_script_content += "r.V_Ray_settings.system_vrayLog_file = ""$vrayLogFile""`r`n"
-        }    
-        ElseIf($renderer -eq "VRayAdv")
-        {
-            $script:pre_render_script_content += "index = findString rendererName ""V_Ray_Adv""`r`n"
-            $script:pre_render_script_content += "if index == 1 then (r.system_distributedRender = true)`r`n"
-            $script:pre_render_script_content += "r.system_vrayLog_level = 4`r`n"
-            $script:pre_render_script_content += "r.system_vrayLog_file = ""$vrayLogFile""`r`n"
```

Comment:
please add this line and the line above back in to the block for maxversion == 2021 and not VRayRT (currently line 136)